test-data/index-x.ts

```diff
@@ -124,7 +124,35 @@ new (class extends TestDataServer {
     } else if (
       (request.url as string) === '/statistics/europeana/target/country/all'
     ) {
-      response.end(JSON.stringify(countryTargetData));
+      const collected = [];
+      // take only the latest from each country
+      const result = countryTargetData
+        .reverse()
+        .filter((item: { country: string }) => {
+          if (collected.includes(item.country)) {
+            return false;
+          } else {
+            collected.push(item.country);
+            return true;
+          }
+        });
+      response.end(JSON.stringify(result));
+    } else if (
+      (request.url as string).includes(
+        '/statistics/europeana/target/country/historical'
```

Comment:
my suggestion is to use constants  for every endpoint (e.g `'/statistics/europeana/targets'`, `'/statistics/europeana/target/country/historical'` )